scripts/cnode-helper-scripts/deploy-as-systemd.sh

```diff
@@ -16,33 +16,13 @@ fi
 echo -e "\e[32m~~ Cardano Node ~~\e[0m"
 echo "launches the main cnode.sh script to start cardano-node"
 echo
-echo "automatically deployed!"
-sudo bash -c "cat << 'EOF' > /etc/systemd/system/${vname}.service
-[Unit]
-Description=Cardano Node
-Wants=network-online.target
-After=network-online.target
+./cnode.sh -d
 
-[Service]
-Type=simple
-Restart=always
-RestartSec=5
-User=$USER
-LimitNOFILE=1048576
-WorkingDirectory=${CNODE_HOME}/scripts
-ExecStart=/bin/bash -l -c \"exec ${CNODE_HOME}/scripts/cnode.sh\"
-ExecStop=/bin/bash -l -c \"exec kill -2 \$(ps -ef | grep ${CNODEBIN}.*.${CNODE_HOME}/ | tr -s ' ' | cut -d ' ' -f2) &>/dev/null\"
-KillSignal=SIGINT
-SuccessExitStatus=143
-StandardOutput=syslog
-StandardError=syslog
-SyslogIdentifier=${vname}
-TimeoutStopSec=5
-KillMode=mixed
+echo -e "\e[32m~~ Cardano DB Sync ~~\e[0m"
+echo "launches the dbsync.sh script to start cardano-db-sync"
+echo
+./dbsync.sh -d
```

Comment:
Doesn't this mean that dbsync service will be deployed for all installations, even simple relays/bp's ?